README.md

```diff
@@ -73,6 +73,56 @@ config.safer_rails_console.warn_text = "WARNING: YOU ARE USING RAILS CONSOLE IN
                                        'Make sure you know what you\'re doing.'
 ```
 
+Configuration settings can also be overridden using environment variables. This can be accomplished
+by instatiating an environment variable prefixed with `SAFER_RAILS_CONSOLE_` and suffixed with an
+uppercased setting name. E.g. `SAFER_RAILS_CONSOLE_WARN_TEXT='Foo: bar'`
+
+Supported values and formatting:
+```bash
+# config.safer_rails_console.environment_names:
+SAFER_RAILS_CONSOLE_ENVIRONMENT_NAMES="{
+    'development' => 'dev',
+    'staging' => 'staging',
+    'production' => 'prod',
+    'foo' => 'bar'
+}"
```

Comment:
are we possibly making this too transparent? It feels like configuring safer_rails_console the way we'll want will be pretty awkward / error prone.

It feels like what we want is a way to:
* override the environment short name to a specific name we want
* override the prompt color
* turn the sandbox/warn on/off via environment variable

Maybe we should just explicitly expose variables to do those things?